shell/platform/android/android_context_vk_impeller.cc

```diff
@@ -68,7 +68,7 @@ static std::shared_ptr<impeller::Context> CreateImpellerContext(
       FML_LOG(IMPORTANT) << "Using the Impeller rendering backend (Vulkan).";
     }
   }
-  if (context->GetDriverInfo()->IsKnownBadDriver()) {
+  if (context && context->GetDriverInfo()->IsKnownBadDriver()) {
```

Comment:
Is there any way to write a test so that the context here comes from the mock context builder?

Reply:
I don't think this code can run outside of Android, mock context or not. I would argue this is tested by way of the failing test that I'm going to fix with this.